src/satosa/context.py

```diff
@@ -60,3 +61,25 @@ def path(self, p):
         elif p.startswith('/'):
             raise ValueError("path can't start with '/'")
         self._path = p
+
+    def decorate_context(self, key, value):
+        """
+        Add information to the context
+
+        :type context: satosa.context.Context
+        :rtype: None
+        :param context: The current context
+        :return: None
+        """
+
+        # Add the metadata store the backend is using to the context so that 
```

Comment:
I think this documention is too specific for the generalized decorator (but I get what you try to say). Maybe wrap your usecase in an 'e.g.'?